packages/katello/katello/hostname-change.rb

```diff
@@ -143,8 +143,7 @@ def next_steps_message
   You will have to install the new bootstrap rpm and reregister all clients and #{@plural_proxy} with subscription-manager
   (update organization and environment arguments appropriately):
 
-  yum remove -y katello-ca-consumer*
-  rpm -Uvh http://#{@new_hostname}/pub/katello-ca-consumer-latest.noarch.rpm
+  set -o pipefail && curl -sS "http://#{@new_hostname}/unattended/public/foreman-ca-refresh" | bash
```

Comment:
Is this what you want?

```suggestion
  curl -sS "http://#{@new_hostname}/unattended/public/foreman-ca-refresh" | bash -o pipefail
```

Reply:
No. When curl fails, we want  to stop bash from running, otherwise the return code of the action would be `0`.

Reply:
When the hostname changes on the server, we don't need to perform a CA refresh, we need to perform a sub-man configuration update (to point to the new hostname).

Reply:
@ehelms I take your point about changing from re-registration to reconfiguration.
Now about hostname change: is there a chance that hostname change also requires change to a certificate? For example if the CA is self-signed?

Reply:
Not the CA certificate since it is not tied to a hostname.

Reply:
I guess, if its only about changing the subscription-manager configuration, something like this is sufficient: https://github.com/theforeman/foreman/blob/develop/app/views/unattended/provisioning_templates/snippet/subscription_manager_setup.erb#L108 ?

Then we should also get rid of https://github.com/theforeman/foreman-packaging/blob/708b947c6bbfe504536aad5d7d70eda34de1a083/packages/katello/katello/hostname-change.rb#L147 and rewrite https://github.com/theforeman/foreman-packaging/blob/708b947c6bbfe504536aad5d7d70eda34de1a083/packages/katello/katello/hostname-change.rb#L149